docker-compose.yml

```diff
@@ -357,6 +357,19 @@ services:
       resources:
         ipv4_address: 172.20.0.100
 
+  download.httpbin: # Named after `httpbin` because that is how DNS resources are configured for the test setup.
+    build:
+      target: dev
+      context: rust
+      dockerfile: Dockerfile
+      args:
+        PACKAGE: http-test-server
```

Comment:
Need to decide whether we want to build this on-demand every time we run the tests or build is part of CI? Maybe it should not even be part of the mono repo? I don't know :shrug: